docs/src/getting-started/basics.md

```diff
@@ -0,0 +1,13 @@
+## Basic usage of the Fuel Indexer
+
+### Instantiating a Fuel client
+Start a local Fuel node in the background
+```bash
```

Comment:
Install markdownlint so you can automatically fix linting errors like this.
````suggestion
Start a local Fuel node in the background

```bash
````

Reply:
@adlerjohn `markdownlint` helped quite a bit. Thanks for the tip 👌